wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,589 @@
+---
+tags:
+- UK Community Cup 7
+- UKCC 7
+- UKCC 7
+---
+
+# UK Community Cup 7
+
+![UKCC 7 Logo](img/banner.png)
+
+The 7th edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th October 2019.
+
+# Contents
+
+[Introduction](#introduction)  
+[Links](#links)  
+[Rules](#rules)  
+[Schedule](#tournament-schedule)  
+[Prizes](#prizes)  
+[Organisation](#organisation)  
+[Players](#players)  
+[Results](#results)  
+[Map pools](#map-pools)  
+[Schedules / Match Results](#Schedules-/-Match-Results)
+
+---
+
+## Introduction
+
+![Introduction](img/intro.png)
+
+- This is a **1v1, UK and Ireland based tournament**
+- In the event there is 128 signups, those with the lowest global rank will be unable to participate (unless we expand the tournament)
+- The structure of the tournament will be:
+    - Two qualifications rounds - 32 players progress
```

Comment:
```suggestion
    - Two qualifications rounds - 32 players progress
```
Line it up right below the first word of previous line. (Use tab)
Do for others too.

Reply:
I do use tab - but my editor is set to 4 space tab. Updated the doc to be 2 space.